projects/core/src/error-handling/cx-error-handler.ts

```diff
@@ -5,13 +5,33 @@
  */
 
 import { ErrorHandler, Injectable, inject } from '@angular/core';
-import { LoggerService } from '../logger/logger.service';
+import { LoggerService } from '../logger';
+import { MULTI_ERROR_HANDLER } from './multi-error-handler';
 
+/**
+ * The CxErrorHandler is the default ErrorHandler for Spartacus.
+ * It is responsible for handling errors and passing them to the registered multi error handlers.
+ *
+ * @method handleError - Handles the error by passing it to the registered multi error handlers.
+ *
+ * @public
+ */
 @Injectable()
 export class CxErrorHandler implements ErrorHandler {
-  logger = inject(LoggerService);
+  //TODO: Keep it updated with the latest version of Spartacus
+  /**
+   * @deprecated Since 6.6 - `LoggerService` is unused here. Instead it's now used in `LoggingErrorHandler`.
```

Comment:
[MEDIUM]
Is this JSDoc outdated?

```suggestion
   * @deprecated Since 2211.29 - `LoggerService` is not used anymore in this class.
   *             Instead it's now used in `LoggingErrorHandler`.
   *             This property will be removed in the future together with removing
   *             the feature toggle `propagateErrorsToServer`.
```